test/Feature/MakeSymbolicAPI.c

```diff
@@ -0,0 +1,21 @@
+// RUN: %llvmgcc %s -emit-llvm -g -c -o %t1.bc
+// RUN: rm -rf %t.klee-out
+// RUN: not %klee --output-dir=%t.klee-out %t1.bc
+// RUN: FileCheck %s --input-file=%t.klee-out/warnings.txt
+
+int main() {
+  unsigned a, b, c, d;
+
+  klee_make_symbolic(&a, sizeof(a), "");
+// CHECK-NOT: KLEE: WARNING: unable to write output test case, losing it
```

Comment:
As I mentioned yesterday, the problem is that the test case is generated at the end of the execution. Unfortunately, the execution is terminated with an error and the tests are not generated. Therefore, this check cannot be performed correctly.
The only way I see is to split the test in two: the good cases are part of `Feature`, while the bad case is part of `regression`.